app/views/posts/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<section class="container mt-5">
+  <div class="row justify-content-center align-items-center "> 
+    <div class="col-6 border border-dark rounded-1  border-2 p-5">
+    <h1 class="text-center">Add new post</h1>
+       <% if flash[:alert].present? %>
+        <div class="alert alert-danger">
+          <%= flash[:alert] %>
+        </div>
+      <% end %>
+       <% if flash[:notice].present? %>
+        <div class="alert alert-success">
+          <%= flash[:notice] %>
+        </div>
+      <% end %>
+      <%= form_with(model: @post, url: user_posts_path) do |form| %>
+        <div class="form-group">
+          <%= form.label :title %>
+          <%= form.text_field :title, class: 'form-control' %>
+        </div>
+
+        <div class="form-group">
+          <%= form.label :text %>
+          <%= form.text_area :text, class: 'form-control' %>
+        </div>
+
+        <%= form.hidden_field :author_id, value: current_user.id %>
+
+        <div class="form-group mt-2">
+          <%= form.submit 'Create Post', class: 'btn btn-primary' %>
```

Comment:
- [ ] While your code appears well written, this form does not submit, when I added data, it failed to submit it and as a result, I could not check the comment form aspect. Kindly double-check it.
![image](https://github.com/siddghosh108/blog-app/assets/86200356/ed20c1a9-d47b-4d36-bb0a-b104a28278e2)

Reply:
I have fixed the error and you can succesfully post now
![Uploading Screenshot 2023-12-14 at 2.53.38 PM.png…]()